### PR TITLE
feat(comp:rate): add color porp

### PR DESCRIPTION
**Pull request by kovsu**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [our guidelines](https://github.com/IDuxFE/idux/blob/main/packages/site/src/docs/Contributing.zh.md#commit)
- [ ] Tests for the changes have been added/updated or not needed
- [x] Docs and demo have been added/updated or not needed

## What is the current behavior?

image

rate 组件只有一个颜色，可以提供一个 color 属性，去设置它的颜色

## What is the new behavior?
image

### CODE REVIEW

File: packages/components/rate/src/types.ts [hunk 1]
```diff
@@ -30,6 +30,10 @@ export const rateProps = {
     default: (): string[] => [],
   },
   size: String as PropType<FormSize>,
+  color: {
+    type: String,
+    default: '#F8D81A',
```
Comment (danranVm) [inline packages/components/rate/src/types.ts:35]:
```suggestion
    default: undefined
```
默认直给 undefined 就行，默认情况下还是用 css 去控制。

**Review by danranVm — CHANGES_REQUESTED** [review]

File: packages/components/rate/src/types.ts (hunk 1 above)
Comment (danranVm) [reply packages/components/rate/src/types.ts:35]: 文档那里也一并改下。

Comment (kovsu) [reply packages/components/rate/src/types.ts:35]: 好的，我在弄。我在琢磨这个 git base。

**Comment by kovsu** [comment]
image
我这里只有一个 提交信息，为什么这里有这么多啊

**Pull request closed without merging**